scanner/mappers/mappers_test.go

```diff
@@ -705,11 +706,106 @@ func Test_toProtoV4VulnerabilitiesMap(t *testing.T) {
 				},
 			},
 		},
+		"when severity and unknown distribution then populate the proto": {
+			ccVulnerabilities: map[string]*claircore.Vulnerability{
+				"foo": {
+					Issued:   now,
+					Severity: "sample severity",
+				},
+			},
+			want: map[string]*v4.VulnerabilityReport_Vulnerability{
+				"foo": {
+					Issued:   protoNow,
+					Severity: "sample severity",
+				},
+			},
+		},
+		"when severity with CVSSv3 and RHEL then find encoded CVSS scores": {
+			ccVulnerabilities: map[string]*claircore.Vulnerability{
+				"foo": {
+					Issued: now,
+					Severity: url.Values{
+						"severity":     []string{"sample severity"},
+						"cvss3_vector": []string{"sample cvss3 vector"},
+						"cvss3_score":  []string{"9.9"},
+					}.Encode(),
+					Dist: &claircore.Distribution{DID: "rhel"},
+				},
+			},
+			want: map[string]*v4.VulnerabilityReport_Vulnerability{
+				"foo": {
+					Issued:   protoNow,
+					Severity: "sample severity",
+					Cvss: &v4.VulnerabilityReport_Vulnerability_CVSS{
+						V3: &v4.VulnerabilityReport_Vulnerability_CVSS_V3{
+							BaseScore: 9.9,
+							Vector:    "sample cvss3 vector",
```

Comment:
We should consider if we want to just trust the vector is formatted as expected or check it ourselves. I can't recall what we do elsewhere

Reply:
258d2c9281